support-workers/src/test/scala/com/gu/zuora/ZuoraITSpec.scala

```diff
@@ -39,6 +40,24 @@ class ZuoraITSpec extends AsyncFlatSpec with Matchers {
     }
   }
 
+  it should "retrieve subscription redemption information from a redemption code" in {
+    val redemptionCode = "gd12-fnc4ox"
```

Comment:
might be worth calling it gd12-autotest so that people don't try to use (or make it obvious some other way)